source/ranges.tex

```diff
@@ -13529,7 +13529,7 @@
     using @\exposid{Base}@ = @\exposid{maybe-const}@<Const, V>;                 // \expos
     iterator_t<@\exposid{Base}@> @\exposid{current_}@   = iterator_t<@\exposid{Base}@>();   // \expos
     iterator_t<@\exposid{Base}@> @\exposid{last_ele_}@  = iterator_t<@\exposid{Base}@>();   // \expos,
-                                               // present only if \tcode{\exposid{Base}} models \tcode{\exposconcept{slide-caches-first}}
+                                               // present only if \exposid{Base} models \tcode{\exposconcept{slide-caches-first}}
```

Comment:
```suggestion
                                               // present only if \exposid{Base} models \exposconcept{slide-caches-first}
```

Reply:
See also <https://github.com/cplusplus/draft/pull/5249#discussion_r799926289>.
@jensmaurer said "The font isn't quiite right otherwise, I think."
I replied
> Perhaps that's worth a separate editorial issue. My suggestion seems to be the norm, where macros that internally use `\tcode` are not redundantly wrapped in `\tcode`.